components/org.wso2.carbon.status.dashboard.core/pom.xml

```diff
@@ -185,8 +185,9 @@
     </build>
     <properties>
         <export.package>
-            !org.wso2.carbon.status.dashboard.core.internal*
-            org.wso2.carbon.status.dashboard.core.*;version="${project.version}"
+            !org.wso2.carbon.status.dashboard.core.internal*,
```

Comment:
I don't see anything wrong in `org.wso2.carbon.status.dashboard.core.*;version="${project.version}"`

@mohanvive Can you explain what you are trying to fix here?